taichi/common/core.h

```diff
@@ -264,7 +264,7 @@ TI_NAMESPACE_END
 //                           Meta-programming
 //******************************************************************************
 
-#include "meta.h"
+#include "taichi/util/meta.h"
```

Comment:
Good, don't forget to remove these stuff.

Reply:
Thanks. As mentioned above these will be done in a later PR so that this one has only file movements.